cluster/gce/configure-vm.sh

```diff
@@ -639,6 +644,28 @@ function run-salt() {
   salt-call --local state.highstate || true
 }
 
+function wait-for-cluster-healthy-or-bust() {
+  local attempt=0
+  local healthy_node_count=0
+  local total_node_count=0
+  echo "== waiting for cluster to be healthy =="
+  while true; do
+    healthy_node_count=$(kubectl --kubeconfig=/var/lib/kubelet/kubeconfig get nodes | grep -v master | awk '{print $3}' | grep -c Ready)
```

Comment:
this adds a dependency on kubectl being installed and on the path. would it be insane to make this query with curl instead?